bluepysnap/circuit_validation.py

```diff
@@ -233,18 +233,21 @@ def _check_rotations():
     _check_rotations()
     components = config['components']
     errors += _check_components_dir('morphologies_dir', components)
-    errors += _check_components_dir('mechanisms_dir', components)
     errors += _check_components_dir('biophysical_neuron_models_dir', components)
     if errors:
         return errors
+    library = group['@library']
```

Comment:
library does not necessary exist in the files.

Reply:
Thank you, I missed it.